pr-test.yml

```diff
@@ -0,0 +1,41 @@
+name: PR Test
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [opened, synchronize]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the code
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Cache yarn dependencies to improve speed
+      - name: Cache yarn dependencies
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      # Install dependencies
+      - name: Install dependencies
+        run: yarn install
```

Comment:
Could you please check if it is working correctly with yarn v4?
My concern is that we might need to enable corepack explicitly.

Reply:
Noted, I will work on these changes